packages/editor/src/components/colors/utils.js

```diff
@@ -41,41 +57,27 @@ export const getColorValue = ( colors, definedColor, customColor ) => {
 * @return {?string} If colorValue is defined and matches a color part of the colors array, it returns the color name for that color.
 */
 export const getColorName = ( colors, colorValue ) => {
+	deprecated( 'getColorName function', {
```

Comment:
Given that we now have a function that returns the color object, I think having a function specific to one of the object properties is now redundat as people can just get the color object and use the name property.

For the Font Size picker case, given the font size picker UI the objects set smallName: (S, M...) and name: (small, medium ...). It would be strange to have a getter for each prop (getFontSizeSmallName, getFontSizeName...) and I preferred to just return the object.
I feel similar components to this one (and FontSize) will be added in the future (by us or plugin developers internally) for paddings, margins etc... Basically in every case where we want to allow the user to set a value but use a class instead if it is a predefined value. We are not aware of all the API properties objects representing predefined values will contain if possible the APIs should be identical. So I think we should just have a function that returns the object everywhere.